ui/pages/groups/index.js

```diff
@@ -32,6 +32,7 @@ function AddGroupsDialog({ setOpen }) {
       router.replace(`/groups/${group.id}`)
     } catch (e) {
       setError(e.message)
+      setName(name.trim())
```

Comment:
If we hit an error we should leave the input fields unchanged – otherwise it's a strange experience!

Reply:
I would normally agree but if left unchanged, the input value will be different than what's being sent in the request.